### PR TITLE
Add Chrome enterprise policies

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This change adds new autoplay related enterprise policies to the "Autoplay Policy Changes" article.

### CODE REVIEW

File: src/content/en/updates/2017/09/autoplay-policy-changes.md [hunk 1]
```diff
@@ -132,6 +132,17 @@ get to the specific blog, so autoplay is allowed. However, the blog needs to
...
 autoplay.
 
+### Chrome enterprise policies
+
+It is possible to change this new autoplay behaviour with Chrome enterprise
+policies for use cases such as kiosks or unattended systems. Check out the
+[Configure policies and settings] help page to learn how to set these new
+autoplay related enterprise policies:
+
+- The ["AutoplayAllowed"] policy controls whether autoplay is allowed or not.
```
Comment (petele) [inline src/content/en/updates/2017/09/autoplay-policy-changes.md]: Should these be backticked (`) instead of quoted?

Comment (beaufortfrancois) [reply src/content/en/updates/2017/09/autoplay-policy-changes.md]: Policies are not `backticked` usually. See http://dev.chromium.org/administrators/policy-list-3#AutoplayAllowed and `chrome://policy`

**Comment by beaufortfrancois** [comment]
@petele Please merge and push live. Thanks!

File: src/content/en/updates/2017/09/autoplay-policy-changes.md (hunk 1 above)
Comment (jpmedley) [reply src/content/en/updates/2017/09/autoplay-policy-changes.md]: Is this invoked in code, or in a user interface? I suspect this should be both. `"AutoplayAllowed"`

Comment (jpmedley) [reply src/content/en/updates/2017/09/autoplay-policy-changes.md]: Then it's quotes, not backticks.

**Pull request merged**